hardware/index.md

```diff
@@ -219,15 +219,27 @@ Power to the micro:bit may be provided via 5V on the USB connector, or via a 3V
 
 The interface chip handles the USB connection, and is used for flashing new code to the micro:bit, sending and receiving serial data back and forth to your main computer.
 
+The V2.2X devices use two interface chip variants due to availability of the parts; 400K devices were built using nRF52820-QDAA and further devices are built using nRF52820-QDAA.
```

Comment:
There's a problem with the issue. One of the interface chips should be ` nRF52833-QDAA`, but both listed here are `nRF52820-QDAA`. I don't know which one was used for 400k builds.